wagtail/admin/templates/wagtailadmin/shared/last_updated.html

```diff
@@ -1,6 +0,0 @@
-{% load i18n wagtailadmin_tags %}
```

Comment:
suggestion: add a note to the upgrade considerations for people to use `{% human_readable_date the_timestamp %}` instead of this include

Reply:
Done